### PR TITLE
feat(core): include turbo into well known jsonc

### PR DESCRIPTION
**Pull request by Netail**
## Summary

Add turbo.json to well known jsonc files.

Also remove yaml alias from error as aliases of other languages aren't included either

### CODE REVIEW

**Comment by ematipico** [comment]
I don't think this should be considered a fix. It should be considered a feature.

Plus:
- you should add a changelog entry
- send a PR to update the documentation: https://biomejs.dev/guides/how-biome-works/#json-like-files

**Comment by Netail** [comment]
> I don't think this should be considered a fix. It should be considered a feature.
Is the "fix:" commit prefix or "fix/" branch prefix used for ci/cd? In case I have to change that

**Comment by ematipico** [comment]
Just the title of the PR :) swap `fix:` with `feat(core):`

**Comment by Netail** [comment]
> send a PR to update the documentation
https://github.com/biomejs/website/pull/686

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,9 +11,10 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 
 ## Unreleased
 
+- Feature [#3306](https://github.com/biomejs/biome/pull/3306) support comments in `turbo.json`.
```
Comment (Sec-ant) [inline CHANGELOG.md:14]: Feel free to add a `Contributed by` note

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,9 +11,10 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
...
 - Implement [css suppression action](https://github.com/biomejs/biome/issues/3278). Contributed by @togami2864
 
-## v1.8.3 (2022-06-27)
+## v1.8.3 (2024-06-27)
```
Comment (Sec-ant) [inline CHANGELOG.md:17]: 👍

File: CHANGELOG.md [hunk 3]
```diff
@@ -738,13 +750,15 @@ z.object({})
     1 1 │   <>
     2   │ - → <div·class={`${variable}·px-2·foo·p-4·bar`}/>
       2 │ + → <div·class={`${variable}·foo·bar·p-4·px-2`}/>
-    3 3 │   	<div class={`px-2 foo p-4 bar ${variable}`}/>
+    3 3 │    <div class={`px-2 foo p-4 bar ${variable}`}/>
```
Comment (Sec-ant) [inline CHANGELOG.md:753]: Is the changelog formatted by some tools? Most of the new lines are okay, but I'm not sure if we want to change these indentations. Maybe you can just leave the formatting as is and only add the relative changelog entry.

**Review by Sec-ant — APPROVED** [review]
Thanks! But I'm not sure about the reformatting of the changelog file. It's good to have a consistent style, but maybe we shouldn't touch some of the indentations.

File: CHANGELOG.md (hunk 3 above)
Comment (Netail) [reply CHANGELOG.md]: Reverted the formatting :)

**Comment by Netail** [comment]
Applied feedback :)

File: CHANGELOG.md [hunk 4]
```diff
@@ -16,6 +16,7 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 #### Enhancements
 
 - Implement [css suppression action](https://github.com/biomejs/biome/issues/3278). Contributed by @togami2864
+- Feature [#3306](https://github.com/biomejs/biome/pull/3306) support comments in `turbo.json`. Contributed by @Netail
```
Comment (ematipico) [inline CHANGELOG.md:19]:
```suggestion
- Add support of comments in `turbo.json`. Contributed by @Netail
```

**Review by ematipico — APPROVED** [review]

**Pull request merged**